addons/account/views/res_config_settings_views.xml

```diff
@@ -386,6 +386,18 @@
                                     </div>
                                 </div>
                             </div>
+                            <div class="col-12 col-lg-6 o_setting_box" title="If you check this box, you will be able to register your payment using ACH.">
+                                <div class="o_setting_left_pane">
+                                    <field name="module_account_dwolla" widget="upgrade_boolean"/>
+                                </div>
+                                <div class="o_setting_right_pane" name="dwolla_right_pane">
+                                    <label for="module_account_dwolla"/>
+                                    <span class="fa fa-lg fa-building-o" title="Values set here are company-specific." aria-label="Values set here are company-specific." groups="base.group_multi_company" role="img"/>
+                                    <div class="text-muted">
+                                        Pay your bills in one-click using ACH (Automated Clearing House, only used in the US)
```

Comment:
"Pay your bills in one-click using ACH (US only)".
Short is better. The longer version is in the tooltip.

Reply:
Done